src/components/Network/AssetList.tsx

```diff
@@ -77,7 +78,7 @@ const AssetList = ({
             type={"button"}
             className={clsx(
               "flex justify-between items-center gap-3 p-2.5 rounded-md hover:bg-gray-950 dark:hover:bg-black-950",
-              isNotSelectable && "opacity-50 pointer-events-none"
```

Comment:
We should start rewriting all Tailwind styles to CSS Modules to avoid Tailwind collisions between the SDK and the frontend.